blog_system/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from django.urls import path, include
+from debug_toolbar.toolbar import debug_toolbar_urls
```

Comment:
It seems like you're trying to import `debug_toolbar_urls` from `debug_toolbar.toolbar`, but this is not the correct import path. You should import it from `debug_toolbar.urls` instead.